test/helpers/fixture_deploy_helper.rb

```diff
@@ -82,19 +82,19 @@ def deploy_dir_without_profiling(dir, wait: true, allow_protected_ns: false, pru
     )
   end
 
-  # Deploys all fixtures in the given directory via KubernetesDeploy::DeployTask
+  # Deploys all fixtures in the given directories via KubernetesDeploy::DeployTask
   # Exposed for direct use only when deploy_fixtures cannot be used because the template cannot be loaded pre-deploy,
   # for example because it contains an intentional syntax error
-  def deploy_dir(dir, **args)
+  def deploy_dir(*dirs, **args)
```

Comment:
Multiple template directories are exceptional, so it seemed more straightforward to change the signature to accept an arbitrary number of `dirs` as opposed to enforcing an actual array. I can do the latter if that's what we ultimately want.